docs/_posts/examples/.eslintrc

```diff
@@ -5,7 +5,8 @@
     "MapboxGeocoder": true,
     "MapboxDirections": true,
     "turf": true,
-    "d3": true
+    "d3": true,
+    "Uint8Array": true
```

Comment:
We need to continue to write examples without using ES6 features such as `let`/`const` or arrow functions (not supported by iOS 9) and `for...of` (not supported by IE 11). I think it's useful to keep this configuration option to catch accidental uses of these types of features. I suggest we revert the latest commit.